docs/usage/self-hosted-configuration.md

```diff
@@ -15,6 +15,11 @@ Please also see [Self-Hosted Experimental Options](./self-hosted-experimental.md
 
 ## allowCustomCrateRegistries
 
+## allowExtraConfig
+
+Set to `true` to allow your own config variable.
```

Comment:
These extra variable(**s**) would be unused, as they are not understood by Renovate. I believe it would make sense if the documentation would state that.